 INFO 31 八月 14:44:57,805 -(              MoDBRW.java: 132) - execSQL: select monet_id,type_id from Equipment where monet_id in (select distinct monetid   from user_event  where server_date > ? and server_date < ? ) =1809
 INFO 31 八月 14:44:58,508 -(              MoDBRW.java: 132) - execSQL: select id,type from EquipmentType =639
 INFO 31 八月 14:45:00,957 -(              MoDBRW.java: 132) - execSQL: select ownerid,petLevel from pet =2449
ERROR 31 八月 14:45:02,019 -(              MoDBRW.java: 229) - execSQLUpdate error
com.microsoft.sqlserver.jdbc.SQLServerException: 到主机  的 TCP/IP 连接失败。 java.net.ConnectException: Connection refused: connect
	at com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDriverError(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.connectHelper(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.loginWithoutFailover(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.connect(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:46)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:290)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:771)
	at org.apache.commons.dbcp.PoolingDriver.connect(PoolingDriver.java:175)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at util.MoDB.getConnection(MoDB.java:54)
	at util.MoDBRW.execSQLUpdate(MoDBRW.java:219)
	at petEquiment.PetEquiment.insertinto(PetEquiment.java:221)
	at petEquiment.PetEquiment.insertEquipment(PetEquiment.java:207)
	at petEquiment.PetEquiment.main(PetEquiment.java:230)
 INFO 31 八月 14:45:02,035 -(              MoDBRW.java: 234) - execSQLUpdate: insert into EquipmentInventory(petLevel,equimentType,equimentCount,equimentPerson,rdate,isAction)values(?,?,?,?,?,?) =1062
ERROR 31 八月 14:45:03,050 -(              MoDBRW.java: 229) - execSQLUpdate error
com.microsoft.sqlserver.jdbc.SQLServerException: 到主机  的 TCP/IP 连接失败。 java.net.ConnectException: Connection refused: connect
	at com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDriverError(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.connectHelper(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.loginWithoutFailover(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.connect(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:46)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:290)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:771)
	at org.apache.commons.dbcp.PoolingDriver.connect(PoolingDriver.java:175)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at util.MoDB.getConnection(MoDB.java:54)
	at util.MoDBRW.execSQLUpdate(MoDBRW.java:219)
	at petEquiment.PetEquiment.insertinto(PetEquiment.java:221)
	at petEquiment.PetEquiment.insertEquipment(PetEquiment.java:207)
	at petEquiment.PetEquiment.main(PetEquiment.java:230)
 INFO 31 八月 14:45:03,050 -(              MoDBRW.java: 234) - execSQLUpdate: insert into EquipmentInventory(petLevel,equimentType,equimentCount,equimentPerson,rdate,isAction)values(?,?,?,?,?,?) =1015
ERROR 31 八月 14:45:04,076 -(              MoDBRW.java: 229) - execSQLUpdate error
com.microsoft.sqlserver.jdbc.SQLServerException: 到主机  的 TCP/IP 连接失败。 java.net.ConnectException: Connection refused: connect
	at com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDriverError(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.connectHelper(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.loginWithoutFailover(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.connect(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:46)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:290)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:771)
	at org.apache.commons.dbcp.PoolingDriver.connect(PoolingDriver.java:175)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at util.MoDB.getConnection(MoDB.java:54)
	at util.MoDBRW.execSQLUpdate(MoDBRW.java:219)
	at petEquiment.PetEquiment.insertinto(PetEquiment.java:221)
	at petEquiment.PetEquiment.insertEquipment(PetEquiment.java:207)
	at petEquiment.PetEquiment.main(PetEquiment.java:230)
 INFO 31 八月 14:45:04,076 -(              MoDBRW.java: 234) - execSQLUpdate: insert into EquipmentInventory(petLevel,equimentType,equimentCount,equimentPerson,rdate,isAction)values(?,?,?,?,?,?) =1026
ERROR 31 八月 14:45:05,091 -(              MoDBRW.java: 229) - execSQLUpdate error
com.microsoft.sqlserver.jdbc.SQLServerException: 到主机  的 TCP/IP 连接失败。 java.net.ConnectException: Connection refused: connect
	at com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDriverError(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.connectHelper(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.loginWithoutFailover(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.connect(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:46)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:290)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:771)
	at org.apache.commons.dbcp.PoolingDriver.connect(PoolingDriver.java:175)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at util.MoDB.getConnection(MoDB.java:54)
	at util.MoDBRW.execSQLUpdate(MoDBRW.java:219)
	at petEquiment.PetEquiment.insertinto(PetEquiment.java:221)
	at petEquiment.PetEquiment.insertEquipment(PetEquiment.java:207)
	at petEquiment.PetEquiment.main(PetEquiment.java:230)
 INFO 31 八月 14:45:05,091 -(              MoDBRW.java: 234) - execSQLUpdate: insert into EquipmentInventory(petLevel,equimentType,equimentCount,equimentPerson,rdate,isAction)values(?,?,?,?,?,?) =1015
ERROR 31 八月 14:45:06,121 -(              MoDBRW.java: 229) - execSQLUpdate error
com.microsoft.sqlserver.jdbc.SQLServerException: 到主机  的 TCP/IP 连接失败。 java.net.ConnectException: Connection refused: connect
	at com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDriverError(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.connectHelper(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.loginWithoutFailover(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.connect(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:46)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:290)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:771)
	at org.apache.commons.dbcp.PoolingDriver.connect(PoolingDriver.java:175)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at util.MoDB.getConnection(MoDB.java:54)
	at util.MoDBRW.execSQLUpdate(MoDBRW.java:219)
	at petEquiment.PetEquiment.insertinto(PetEquiment.java:221)
	at petEquiment.PetEquiment.insertEquipment(PetEquiment.java:207)
	at petEquiment.PetEquiment.main(PetEquiment.java:230)
 INFO 31 八月 14:45:06,121 -(              MoDBRW.java: 234) - execSQLUpdate: insert into EquipmentInventory(petLevel,equimentType,equimentCount,equimentPerson,rdate,isAction)values(?,?,?,?,?,?) =1030
ERROR 31 八月 14:45:07,152 -(              MoDBRW.java: 229) - execSQLUpdate error
com.microsoft.sqlserver.jdbc.SQLServerException: 到主机  的 TCP/IP 连接失败。 java.net.ConnectException: Connection refused: connect
	at com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDriverError(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.connectHelper(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.loginWithoutFailover(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.connect(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:46)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:290)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:771)
	at org.apache.commons.dbcp.PoolingDriver.connect(PoolingDriver.java:175)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at util.MoDB.getConnection(MoDB.java:54)
	at util.MoDBRW.execSQLUpdate(MoDBRW.java:219)
	at petEquiment.PetEquiment.insertinto(PetEquiment.java:221)
	at petEquiment.PetEquiment.insertEquipment(PetEquiment.java:207)
	at petEquiment.PetEquiment.main(PetEquiment.java:230)
 INFO 31 八月 14:45:07,152 -(              MoDBRW.java: 234) - execSQLUpdate: insert into EquipmentInventory(petLevel,equimentType,equimentCount,equimentPerson,rdate,isAction)values(?,?,?,?,?,?) =1031
ERROR 31 八月 14:45:08,183 -(              MoDBRW.java: 229) - execSQLUpdate error
com.microsoft.sqlserver.jdbc.SQLServerException: 到主机  的 TCP/IP 连接失败。 java.net.ConnectException: Connection refused: connect
	at com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDriverError(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.connectHelper(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.loginWithoutFailover(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.connect(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:46)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:290)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:771)
	at org.apache.commons.dbcp.PoolingDriver.connect(PoolingDriver.java:175)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at util.MoDB.getConnection(MoDB.java:54)
	at util.MoDBRW.execSQLUpdate(MoDBRW.java:219)
	at petEquiment.PetEquiment.insertinto(PetEquiment.java:221)
	at petEquiment.PetEquiment.insertEquipment(PetEquiment.java:207)
	at petEquiment.PetEquiment.main(PetEquiment.java:230)
 INFO 31 八月 14:45:08,183 -(              MoDBRW.java: 234) - execSQLUpdate: insert into EquipmentInventory(petLevel,equimentType,equimentCount,equimentPerson,rdate,isAction)values(?,?,?,?,?,?) =1031
ERROR 31 八月 14:45:09,198 -(              MoDBRW.java: 229) - execSQLUpdate error
com.microsoft.sqlserver.jdbc.SQLServerException: 到主机  的 TCP/IP 连接失败。 java.net.ConnectException: Connection refused: connect
	at com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDriverError(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.connectHelper(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.loginWithoutFailover(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.connect(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:46)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:290)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:771)
	at org.apache.commons.dbcp.PoolingDriver.connect(PoolingDriver.java:175)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at util.MoDB.getConnection(MoDB.java:54)
	at util.MoDBRW.execSQLUpdate(MoDBRW.java:219)
	at petEquiment.PetEquiment.insertinto(PetEquiment.java:221)
	at petEquiment.PetEquiment.insertEquipment(PetEquiment.java:207)
	at petEquiment.PetEquiment.main(PetEquiment.java:230)
 INFO 31 八月 14:45:09,198 -(              MoDBRW.java: 234) - execSQLUpdate: insert into EquipmentInventory(petLevel,equimentType,equimentCount,equimentPerson,rdate,isAction)values(?,?,?,?,?,?) =1015
ERROR 31 八月 14:45:10,213 -(              MoDBRW.java: 229) - execSQLUpdate error
com.microsoft.sqlserver.jdbc.SQLServerException: 到主机  的 TCP/IP 连接失败。 java.net.ConnectException: Connection refused: connect
	at com.microsoft.sqlserver.jdbc.SQLServerException.makeFromDriverError(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.connectHelper(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.loginWithoutFailover(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerConnection.connect(Unknown Source)
	at com.microsoft.sqlserver.jdbc.SQLServerDriver.connect(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at org.apache.commons.dbcp.DriverManagerConnectionFactory.createConnection(DriverManagerConnectionFactory.java:46)
	at org.apache.commons.dbcp.PoolableConnectionFactory.makeObject(PoolableConnectionFactory.java:290)
	at org.apache.commons.pool.impl.GenericObjectPool.borrowObject(GenericObjectPool.java:771)
	at org.apache.commons.dbcp.PoolingDriver.connect(PoolingDriver.java:175)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at java.sql.DriverManager.getConnection(Unknown Source)
	at util.MoDB.getConnection(MoDB.java:54)
	at util.MoDBRW.execSQLUpdate(MoDBRW.java:219)
	at petEquiment.PetEquiment.insertinto(PetEquiment.java:221)
	at petEquiment.PetEquiment.insertEquipment(PetEquiment.java:207)
	at petEquiment.PetEquiment.main(PetEquiment.java:230)
 INFO 31 八月 14:45:10,213 -(              MoDBRW.java: 234) - execSQLUpdate: insert into EquipmentInventory(petLevel,equimentType,equimentCount,equimentPerson,rdate,isAction)values(?,?,?,?,?,?) =1015
